.github/ISSUE_TEMPLATE/general-support.yml

```diff
@@ -0,0 +1,24 @@
+name: General support
+description: Suggestions that are not related to bugs or new features
+title: 'auro-icon: [issue]'
+labels:
+  - 'Type: Question'
+  - 'Type: Feature'
+  - not-reviewed
+  - auro-icon
+body:
+  - type: textarea
+    id: request
+    attributes:
+      label: General Support Request
+      description: A clear and concise description of what you are interested in seeing.
+  - type: textarea
+    id: solution
+    attributes:
+      label: Possible Solution
+      description: Not obligatory, but suggest an idea of how to implement the requested update.
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional context
+      description: Add any other context or screenshots about the feature request here.
```

Comment:
insert_final_newline

Reply:
Hi @blackfalcon ,

I will update this file as your feedback thank you.